src/modules/i18n/__mocks__/locales.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I think it would be good to make these mocks shorter. That way the snapshots get shorter.